lib/webtask.js

```diff
@@ -46,7 +46,14 @@ function Webtask (sandbox, token) {
     Object.defineProperty(this, 'url', {
         enumerable: true,
         get: function () {
-            var url = this.sandbox.url + '/api/run/' + this.sandbox.container;
+            var url;
+            if (this.claims.host) {
+               var surl = Url.parse(this.sandbox.url);
+               url = surl.protocol + '//' + this.claims.host + (surl.port ? (':' + surl.port) : '') + '/' + this.sandbox.container;
```

Comment:
Yes. In my /etc/hosts I have 
```
127.0.0.1    serverless.host
```
and when I `wt create hello.js --host serverless.host` against a locally running webtask runtime (listening on 8721) I get
```
http://serverless.host:8721/tjanczuk/hello
```